### DISCUSSION TITLE
What is the report type if I want to download "Inventory reconciliation"

### DISCUSSION THREAD

**Discussion by rohitdobariya** [amzn/selling-partner-api-models#3188] [General]
I want to know what is the report type there in SP API if I want to download the "Inventory reconciliation" report
Here is the header which I downloaded report from Seller central and I want this using SP-API
"sku","fnsku","asin","product-name","condition","beginning-quantity","ending-quantity","received","returned","found","sold","removed","lost","disposed","other","discrepant-quantity"

**Comment by jayallu** [comment]
Inventory ledger detailed/summary reports will help
@rohitdobariya 
https://sellercentral.amazon.com/gp/help/external/4FKT5KQWFFJ7LDN
